lib/rbs/prototype/runtime.rb

```diff
@@ -321,6 +329,11 @@ def generate_class(mod)
         )
 
         each_mixin(mod.included_modules, *mod.superclass.included_modules, *mod.included_modules.flat_map(&:included_modules)) do |included_module|
+          unless included_module.name
+            RBS.logger.warn("Skipping anonymous module #{included_module}")
+            next
+          end
```

Comment:
Should I move this and [unless clause below](https://github.com/ruby/rbs/pull/302/files#diff-d15f6899dad56f0e659ff044e5c71b36R377-R381) to the inside of `def each_mixin`?

Reply:
My opinion is to keep the test here.